### DISCUSSION TITLE
Permisssion Problem

### DISCUSSION THREAD

**Discussion by ghost** [localsend/localsend#646] [Discussion (English, International)]
Hi,
First thanks for the new update. I admire that this is the best file transfer software in my knowledge. I am using MX Linux 23 OS.
Problem:
I ve just updated version 1.11 of LocalSend on **Desktop**.
Now when I try to send file and click on **File or Folder button**, I got the error: "NO PERMISSION. You haven't granted necessary permission. Please grant them in Settings."
Now, in Settings, I don't find where to grant the Permission.
And also, I am unable to change "**Download**" directory.  If I click on Download button in Settings, no Response.

Thanks in advance.

**Comment by myviolinsings** [comment]
Same issue in Mx Linux 23 ahs; I can receive files, but cannot send even though ufw has the rules set. Get the same error about you need to set permissions, but nowhere in settings can you do this.

**Comment by fz68** [comment]
same problem here in swaywm, but OK in KDE plasma

**Comment by Tahi2** [comment]
same problem here in sway on fedora41 but haven't tried other wm os's yet...

**Comment by VorTechnix** [comment]
I'm getting this issue on an older Windows 10 desktop with the latest version (1.17.0) of localsend. However the issue is only when I try to add a folder. Files work fine.

The firewall is set to allow on private networks and public (just in case that was the issue). I tried running both as regular user and as administrator.

It would be helpful if the error message specified what *exactly* the setting was that was causing the problem.

**Comment by shian07** [comment]
Hey, so I found a way, It is due to flutter's picking a file in Linux, so flutter has some problem so it can't get all the permission in Linux, what you can do is drag the file from file manager to the local send ( In my case it worked). I was trying to send a Game file, but even dragging was not working, why ? Cause there was a file in the games folder fully empty without any bytes or permission, I just got inside the game folder and dragged everyother file except that empty file, and it started working.

**Reply by myviolinsings** [reply]
Thanks. I will try that and see if it fixes the issue.
